src/components/NetworkSelectorContainer.js

```diff
@@ -1,9 +1,22 @@
-import React, { Component } from "react";
+import { Component } from "react";
 import PropTypes from "prop-types";
+/** @jsx jsx */
+import { css, jsx } from "@emotion/core";
 import { connect } from "react-redux";
 import { updateWeb3, getNetwork, getCustomRpc } from "../reducers/application";
 import { NETWORK_TYPES } from "../config";
 
+const divBase = css`
+  border-radius: 50px;
+  padding: 0.2em 0.5em;
+`;
+
+const selectorBase = css`
```

Comment:
selectorBase ? or selectBase ?

It looks like css reset, wanna apply it lobally instead ?

Reply:
this is just the base style for the selector

Reply:
Added to Global CSS in Meta.js, please review again.